templates/contracts/js/deploy.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+./build.sh
+
+echo ">> Deploying contract"
+
+# https://docs.near.org/tools/near-cli#near-dev-deploy
+near dev-deploy --wasmFile build/hello_near.wasm
+
+export $(cat ./neardev/dev-account.env)
```

Comment:
Now that I think about it, we could better use `export CONTRACT_NAME=$(cat ./neardev/dev-account)` since it is more explicit in this case